GbrainsComposeProject/app/src/main/java/com/nbgsoftware/gbrains/ui/components/ToolbarComponent.kt

```diff
@@ -0,0 +1,51 @@
+package com.nbgsoftware.gbrains.ui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.nbgsoftware.gbrains.R
+
+@Preview(showBackground = true)
+@Composable
+fun ToolbarComponent() {
```

Comment:
I think we should create method have parameter. For example: resIcon1, textIcon1, resIcon2, textIcon2,....
You should not hard code (hard resource, hard text)